llvm/docs/GettingInvolved.rst

```diff
@@ -110,7 +105,7 @@ Discourse forums. There are also commit mailing lists for all commits to the LLV
 The :doc:`CodeOfConduct` applies to all these forums and mailing lists.
 
 `LLVM Discourse`__
-  The forums for all things LLVM and related sub-projects. There are categories and subcategories for a wide variety of areas within LLVM. You can also view tags or search for a specific topic. 
+  The forums for all things LLVM and related sub-projects. There are categories and subcategories for a wide variety of areas within LLVM. You can also view tags or search for a specific topic.

```

Comment:
There seems to be a lot of these fixups of trailing whitespace. It's probably fine, but mildly distracting when looking for the actual changes. (It would probably be fine to just push a commit that cleans up that whitespace without review.)

Reply:
+1 trailing whitespace and line wraps